Gemfile

```diff
@@ -45,9 +45,11 @@ end
 
 group :test do
   gem 'capybara'
+  gem 'draft_matchers', github: 'jelaniwoods/draft_matchers', branch: 'main'
```

Comment:
I don't remember mentioning it ever before, but let me state that it's great that you created this gem! Should we add it to `base-rails`/the Docker image?

Reply:
Thanks! With the matchers that I wrote so far, I'm not sure it's worth it to include in all projects. Maybe after I finish the matcher for ["label has matching input"](https://github.com/jelaniwoods/draft_matchers/wiki#label-matches-input) b/c that is a big pain to write otherwise.